src/styles/card.scss

```diff
@@ -0,0 +1,83 @@
+%displayRatingAndPrice {
+  display: inherit;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card {
+  $blockName: &;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  height: $card-height;
+  padding: 14px;
+  border: 1px solid $card-brdr-color;
+  border-radius: 5px;
+  background-color: $card-bg-color;
+  box-sizing: border-box;
+
+  &__photo {
+    margin: 16px 0 42px;
+
+    #{$blockName}__img {
+      width: 160px;
+      height: 134px;
+    }
```

Comment:
don't write it inside photo element, you increase specificity this way

Reply:
fixed